server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/visitors/CloudFoundryNodeVisitor.java

```diff
@@ -87,12 +87,16 @@ private void handleStandardLifecycle(RootNode node) {
         for (Operation operation : node.getStandardLifecycle().getOperations()) {
             // artifact path
             if (operation.getArtifact().isPresent()) {
-                myApp.addFilePath(operation.getArtifact().get().getFilePath());
```

Comment:
refactor can be done here with both if clauses

Reply:
I am not sure what exactly you mean, but I add some commands to an external method